rlclientlib/schema/DecisionMode.fbs

```diff
@@ -0,0 +1,16 @@
+namespace reinforcement_learning.messages.flatbuff;
+
+union Mode { OnlineMode, ImitationMode }

```

Comment:
Instead of empty structs use enum + int.  
Please check comment in RankingEvent.fbs

Reply:
Please check comment in RankingEvent.fbs